site/content/deployment/openshift_s2i.md

```diff
@@ -34,6 +34,21 @@ the version of your OpenShift cluster. This guide was tested with
 OpenShift 3.7.2. It should work with older or newer versions as well, but that
 is untested.
 
+{{% warning title="Curl on Mac OS X" %}}
+The `curl` binary on Mac OS X suffers from an issue with TLS SNI. Also see: https://github.com/curl/curl/issues/1533
+
+As the use of SNI is required for Kubernetes/OpenShift when it comes to routing
```

Comment:
My understanding is that this is just an issue on OpenShift when using routes (which do not exist on Kubernetes, do they?)

Reply:
This effects all cases where TLS SNI is used. In the issue mentioned I saw some references to kubernetes as well. As I don't have any hands on experience with plain kubernetes, I don't know. I can remove it, if you think it is better.